portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,39 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="navbar">
+        <ul>
+          <li><a href="index.html">home</a></li> 
+          <li><a href="projects.html">projects</a></li> 
+          <li><a href="blog.html">blog</a></li> 
+          <li><a href="photos.html">photos</a></li> 
+          <li><a href="random.html">random</a></li>
+        </ul>
+    </div> 
+    
+    <div id="content">  
+      <div id = "homecontent">
+        <h1>Catherine Yu's Portfolio</h1>
+        <p>This is my portfolio.</p>
+        <p>Click here to get a random message:</p>
+        <button onclick="addRandomGreeting()">Hello</button>
```

Comment:
OK for this PR, but maybe for a future PR if there's time or add a TODO, or even just for note on this as an fyi for the capstone/future projects:

Having these 'onclick' event listeners coded in the HTML is generally not good practice for separation of concerns --since we want the Javascript behavior (code) to be separate from the HTML presentation (mark-up).

There's frameworks to make this eaiser (ie jQuery to manage event-handlers), but could also be done in raw javascript  (though slightly more messy).

Reply:
Understood! I think this was actually in the starter code for this project...but I will make a note for the future!